cli/cmd/generate_gcp.go

```diff
@@ -63,6 +60,8 @@ var (
 	CachedGcpAssetIacParams                  = "iac-gcp-generate-params"
 	CachedAssetGcpExtraState                 = "iac-gcp-extra-state"
 
+	InvalidProjectIDMessage = "Invalid GCP project ID.  It must be 6 to 30 lowercase ASCII letters, digits, or hyphens. It must start with a letter. Trailing hyphens are prohibited. Example: tokyo-rain-123."
```

Comment:
nit. We should probably remove the `.` from the example
```suggestion
	InvalidProjectIDMessage = "Invalid GCP project ID.  It must be 6 to 30 lowercase ASCII letters, digits, or hyphens. It must start with a letter. Trailing hyphens are prohibited. Example: tokyo-rain-123"
```